package.json

```diff
@@ -2,9 +2,15 @@
   "name": "@github/combobox-nav",
   "description": "Attach combobox navigation behavior to an input.",
   "version": "2.1.5",
-  "main": "dist/index.js",
+  "main": "dist-cjs/index.js",
   "module": "dist/index.js",
-  "type": "module",
```

Comment:
curios if we want to remove this, or if renaming `dist-cjs/index.js` to `dist/cjs/index.cjs` might b another option here.

Reply:
Renaming might be an option, but I don't see a way to do that with `tsc` and I'd rather not introduce another tool. I saw that `primer/react` doesn't have this field so I figured it's _probably_ okay to remove it.